lib/runtime/life/resolver.go

```diff
@@ -19,15 +26,18 @@ import (
 type Resolver struct{} // TODO: move context inside resolver
 
 // ResolveFunc ...
-func (r *Resolver) ResolveFunc(module, field string) exec.FunctionImport {
+func (r *Resolver) ResolveFunc(module, field string) exec.FunctionImport { // nolint
 	switch module {
 	case "env":
 		switch field {
 		case "ext_logging_log_version_1":
+			// todo test
 			return ext_logging_log_version_1
 		case "ext_misc_print_utf8_version_1":
+			// todo test
 			return ext_misc_print_utf8_version_1
 		case "ext_misc_print_hex_version_1":
+			// todo test
```

Comment:
probably don't need to test print functions, they are super hard to test anyways

Reply:
Right, I wasn't sure how I would test these.  Removed todos.